### PR TITLE
Sb/machine translation en de 0002

### PR DESCRIPTION
**Pull request by bes-dev**
updates related to https://github.com/openvinotoolkit/open_model_zoo/issues/2286

### CODE REVIEW

**Comment by bes-dev** [comment]
@vladimir-dudnik please look at PR

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/machine-translation-nar-de-en-0002/accuracy-check.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+models:
+  - name: machine-translation-nar-de-en-0002
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: narnmt
+          vocabulary_file: machine-translation-nar-de-en-0002/tokenizer_tgt/vocab.json
+          merges_file: machine-translation-nar-de-en-0002/tokenizer_tgt/merges.txt
+          output_name: preds
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-de-en-0002/accuracy-check.yml:9]:
```suggestion
          output_name: pred
```

File: models/intel/machine-translation-nar-en-de-0002/accuracy-check.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+models:
+  - name: machine-translation-nar-en-de-0002
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: narnmt
+          vocabulary_file: machine-translation-nar-en-de-0002/tokenizer_tgt/vocab.json
+          merges_file: machine-translation-nar-en-de-0002/tokenizer_tgt/merges.txt
+          output_name: preds
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-en-de-0002/accuracy-check.yml:9]:
```suggestion
          output_name: pred
```

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
23:00:44  + /jenkins/workspace/omz/precommit/documentation/jenkins/omz/scripts/documentation_check.py src/models/intel
23:00:44  machine-translation-nar-en-de-0002 not in index.md file

**Comment by vladimir-dudnik** [comment]
@bes-dev there is merge conflict

**Comment by bes-dev** [comment]
> @bes-dev there is merge conflict

done

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**